Makefile

```diff
@@ -11,13 +11,15 @@ TESSDATA =  $(LOCAL)/share/tessdata
 # Name of the model to be built. Default: $(MODEL_NAME)
 MODEL_NAME = foo
 
+OUTPUT_DIR = data/$(MODEL_NAME)
```

Comment:
Add a comment before and re-run shinclude to have this variable listed in `make` / `make help`?

Reply:
Explicitly exposing the output directory to the user would be an additional new feature. I had not planned that in my commit to keep the user interface restricted to things which are really needed, but of course it can be easily added if desired.

Reply:
> Explicitly exposing the output directory to the user would be an additional new feature. 

I think it is a new feature and useful and users should be made aware of it.